app/src/main/java/com/github/se/eventradar/model/Resource.kt

```diff
@@ -0,0 +1,7 @@
+package com.github.se.eventradar.model
+
+sealed class Resource<out T> {
+  data class Success<out T>(val data: T) : Resource<T>()
+
```

Comment:
It could be beneficial but might be overkill for our purposes.

Reply:
> imp: Do you think a Loading data class could still be beneficial to show user feedback when the data is still being fetched in the asynchronous operation?

This will not work since your functions are `suspend` functions